CMakeLists.txt

```diff
@@ -9,8 +9,15 @@ option(ULTIMATE_LIFE_BUILD_TESTS   "Build google test project" OFF)
 
 # Fix MSVC Static Linking
 if (MSVC)
+    message(STATUS "Building using MSVC")
     set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 endif(MSVC)
+if(UNIX)
+    message(STATUS "Building using CLANG")
+    set(CMAKE_C_LINKER_FLAGS "-std=c11")
+    set(CMAKE_CXX_LINKER_FLAGS "-std=c++11")
```

Comment:
Actually do we need to set these flags? Since we discovered what was actually causing the `std::__cxx11::basic_string...` error we were having. It wasn't this